web/deploy/terraform/modules/bootstrap/state_storage/variables.tf

```diff
@@ -0,0 +1,22 @@
+variable "bucket_name" {
+  description = "The name of the S3 bucket to store Terraform state. Must be globally unique."
+  type        = string
+  default     = "osm-terraform-state-storage"
+}
+
+variable "table_name" {
+  description = "The name of the DynamoDB table. Must be unique in this AWS account."
+  type        = string
+  default     = "terraform-state-locks"
+}
+
+variable "aws_region" {
+  description = "The AWS region used by the deployment"
+  type        = string
+  default     = "us-east-1"
+}
+
+variable "development_environment" {
```

Comment:
@smokestacklightnin I'll leave it up to you, but I'd use `environment` as the variable name for simplicity's sake

Reply:
Done

I guess there's no ambiguity, especially with the variable description.